docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+in a manner defined by the selected predicate. A full list of predicates and their behavior are listed below:
+
+| **Predicate** | **Behavior** |
+| ----------- | ----------- |
+| eguals | Created domain must be equal to the domain. |
+| contains | Created domains must contain the domain in some way. |
+| all_equal | All created domains must be equal to the domain. |
+| all_end_with | All created domains or subdomains must end with the selected domain. |
+| is_subset | All created domains must be a subset of the selected domain. This requires the certificate requested by tbot to be be listed with the role's predicate. |
+
+In the case of an example role for use within this guide, use a text editor of your choice to create a role YAML named `tbotrole.yaml` with the following configuration options, replacing `my.domain.com` with your own primary domain name,
+or the domain associated with your cluster:
+
+```code
+ kind: role
+metadata:
+  name: hostcert-bot
+version: v5
+spec:
+  allow:
+    rules:
+      - resources:
+          - host_cert
+        verbs:
+          - create
+          - list
+          - read
+          - update
+          - delete
+        where: 'all_end_with(host_cert.principals, ".my.domain.com")'
+```
+
+Once complete, create your role to apply to the tbot configuration in the next step:
+
+```code
+$ tctl create tbotrole.yaml
+```
+
+
+### Step 3/4 Create Your `tbot` Configuration
+
+Before you create a bot user, you need to determine which role(s) you want to
+assign to it. You can use the `tctl` command below to examine what roles exist
+on your system.
+
+<ScopedBlock scope={["cloud"]}>
+On your client machine, log in to Teleport using `tsh`, then use `tctl` to examine
+what roles exist on your system.
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+Connect to the Teleport Auth Server and use `tctl` to examine what roles exist on
+your system.
+</ScopedBlock>
+
+```code
+$ tctl get roles --format=text
+```
+
+You will see something like the output below on a fresh install of Teleport with the
+default roles—your cluster may have different roles. In this example, let's
+assume you want to give the bot the `hostcert-bot` role to allow it to generate host certificates.
+
+```
+Role    Allowed to login as                           Node Labels Access to resources
+------- --------------------------------------------- ----------- ----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read
+editor
+hostcert-bot  host_cert:create,list,read,update,delete             user:list,create,read,update,delete,...
+```
+
+Machine ID can join with a `token` or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
```

Comment:
```suggestion
`tbot` can join the Teleport Cluster using a `token` or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
```

We will be adding new join methods for tbot very frequently. I wonder if there's an alternative way to phrase this sentence that doesn't require us to make udpates every time we add a new join method.

Reply:
> I wonder if at some point it'll be easier to refer people to the getting started guide and to amend their existing tbot config rather than setting it up from scratch each time.

I was debating this, but I think that there are enough differences for Host Certificates that it warrants a different getting started style guide similar to what we do with [Application Access](https://goteleport.com/docs/machine-id/guides/applications/)

Down the line I see us consolidating all information into a guide that fully describes major features and configurations in a single resource and revamping our landing page to make that information more well understood upfront, but I also believe basic install guides like this will continue to serve a similar but separate role.